test/js/ui/map.test.js

```diff
@@ -103,7 +98,7 @@ test('Map', function(t) {
             style.fire('source.error');
             style.fire('source.change');
             style.fire('tile.add');
-            style.fire('tile.load');
+            // style.fire('tile.load');
```

Comment:
This is commented out because I discovered that the event isn't actually fired (but that wasn't caught by the test because it fires two change events, the second one after the sprite is loaded). We should fix this test in a separate PR (not trivial because of the asyncness).